docs/data/introduction/licensing/licensing.md

```diff
@@ -281,6 +281,15 @@ However, when the term ends, you are not allowed to use the current or older ver
 
 To solve the issue, you can [renew your license](https://mui.com/r/x-get-license/) or stop making changes to code depending on MUI X's APIs.
 
+:::warning
+Make sure to set `process.env.NODE_ENV` to `'production'` in your build process to avoid the watermark in production.
+Most bundlers set this environment variable automatically when building for production, but for custom setups, you might need to set it manually.
+
+Note that `NODE_ENV=production` is not MUI-specific and is a common practice in the JavaScript ecosystem.
```

Comment:
We almost never use MUI standalone in the docs. Like the React docs almost never mention Meta. A quick fix: 06e1e7284.